File: Codeforces/282A_Bit++.cpp
#include <iostream>

using namespace std;

int main() {
    int n;
    cin >> n;

    int result = 0;

    while (n != 0) {

        string input;
        cin >> input;

        if (input == "++X" || input == "X++")
            result++;
        else if (input == "--X" || input == "X--")
            result--;

        n--;
    }

    cout << result << endl;

    return 0;
}

File: Codeforces/282A_Bit++.py
n = int(input())
x = 0

while n != 0:
    statement = input()
    if statement in ['X++', '++X']:
        x += 1
        n -= 1
    if statement in ['X--', '--X']:
        x -= 1
        n -= 1

print(x)
